src/runtime/cpu_device_api.cc

```diff
@@ -20,13 +20,14 @@ class CPUDeviceAPI final : public DeviceAPI {
       *rv = 1;
     }
   }
-  void* AllocDataSpace(TVMContext ctx, size_t size, size_t alignment) final {
+  void* AllocDataSpace(TVMContext ctx, TVMType type, size_t nbytes,
```

Comment:
if we have multiple line breaks, it might be more natural to break each argument into one line

Reply:
Changed.